src/components/plugins/export/ExportControls.jsx

```diff
@@ -15,44 +15,45 @@ class ExportControls extends Component<Props> {
   }
 
   exportData() {
-    const { fileName, exportData, columnModel } = this.props;
+    const { exportFileName, data, columnModel } = this.props;
     let CSV = '';
     let header = '';
 
-    if (columnModel && columnModel.length && exportData && exportData.length) {
-      columnModel.forEach((element) => {
-        header += `${element.name},`;
-      });
+    columnModel.forEach((element) => {
+      header += `${element.name},`;
+    });
 
-      header = header.slice(0, -1);
-      CSV += `${header}\r\n`;
+    header = header.slice(0, -1);
+    CSV += `${header}\r\n`;
 
-      for (let i = 0; i < exportData.length; i += 1) {
-        let rowData = '';
-        for (let j = 0; j < columnModel.length; j += 1) {
-          rowData += `${exportData[i][columnModel[j].dataIndex]},`;
-        }
-        rowData.slice(0, rowData.length - 1);
-        CSV += `${rowData}\r\n`;
+    for (let i = 0; i < data.length; i += 1) {
+      let rowData = '';
+      for (let j = 0; j < columnModel.length; j += 1) {
+        const value = columnModel[j].getExportValue
```

Comment:
Please update `getExportValue` with `formatter`. `formatter` is already an expected prop on columns, and should do what you're expecting.